### PR TITLE
feat: automate theme installation

### PR DESCRIPTION
**Pull request by orlando**
**What:** automate theme installation

**Why:** Closes https://app.asana.com/0/1168997577035609/1179093817954641

**How:**

- Use `rake themes:install` command to install the theme from Github repo.
- Remove merged themes from install

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,14 +29,15 @@ install:
...
 	git clone https://github.com/debtcollective/discourse-custom-wizard.git ../plugins/discourse-custom-wizard
-	git clone https://github.com/debtcollective/discourse-debtcollective-private-message.git ../plugins/discourse-debtcollective-private-message
 	git clone https://github.com/debtcollective/discourse-debtcollective-sso.git ../plugins/discourse-debtcollective-sso
-	git clone https://github.com/debtcollective/discourse-debtcollective-collectives.git ../plugins/discourse-debtcollective-collectives
 	git clone https://github.com/debtcollective/discourse-sentry.git ../plugins/discourse-sentry
 	git clone https://github.com/debtcollective/discourse-skylight.git ../plugins/discourse-skylight
 	git clone https://github.com/debtcollective/discourse-mailchimp-list.git ../plugins/discourse-mailchimp-list
 	git clone https://github.com/discourse/discourse-adplugin.git ../plugins/discourse-adplugin
 
+	# install theme
```
Comment (arathjz) [inline Makefile:38]: I tested setting up my environment from scratch with this branch and got this error

```
Caused by:
PG::ConnectionBad: FATAL:  database "debtcollective_discourse_development" does not exist
/Users/arathjz/debt-collective/discourse/lib/site_settings/db_provider.rb:61:in `table_exists?'
```

Should the process `install theme` be after setting up the discourse environment?

Comment (duranmla) [reply Makefile:38]: Maybe we can add posgres commands before running this? like:

```
createdb debtcollective_discourse_development
```

Comment (orlando) [reply Makefile:38]: ok I'll move this step after we create the db. Thanks for reporting @arathjz

**Comment by duranmla** [comment]
But we never create the db within the `Makefile` right? and the changes at `d513c67` doesn't explicit address the error from @arathjz or I am failing to realise something else.

**Comment by orlando** [comment]
@duranmla we do it here https://github.com/debtcollective/discourse-import/blob/d513c6705c9891a9a146e958a91ecb5d44e2086f/Makefile#L58

**Pull request merged**